flaky/flaky_pytest_plugin.py

```diff
@@ -171,34 +175,34 @@ def terminal_summary(self, stream):
             self._add_flaky_report(stream)
 
     @staticmethod
-    def _get_test_method_name(test):
+    def _get_test_callable_name(test):
         """
-        Get the name of the test method from the test.
+        Get the name of the test callable from the test.
         :param test:
             The test that has raised an error or succeeded
         :type test:
             :class:`Function`
         :return:
-            The name of the test method that is being run by the test
+            The name of the test callable that is being run by the test
         :rtype:
             `unicode`
         """
         return test.name
```

Comment:
Should be. All the changes to the pytest side are just variable/parameter names. The bug was in nose only.